pkg/apis/submariner.io/v1/types.go

```diff
@@ -84,14 +82,14 @@ type GatewayStatus struct {
 	Connections   []Connection `json:"connections"`
 }
 
-// LatencySpec describes the round trip time information in nanoseconds for a packet
+// LatencySpec describes the round trip time information in miliseconds for a packet
 // between the gateway pods of two clusters.
 type LatencySpec struct {
-	LastRTT    time.Duration `json:"lastRTT"`
-	MinRTT     time.Duration `json:"minRTT"`
-	AverageRTT time.Duration `json:"averageRTT"`
-	MaxRTT     time.Duration `json:"maxRTT"`
-	StdDevRTT  time.Duration `json:"stddevRTT"`
+	LastRTT    float64 `json:"lastRTT"`
```

Comment:
What's the reason for changing these to float and ms units?

Reply:
The value in nano seconds seems to be not understandable, like

 Latency:
      Average RTT:   585160
      Last RTT:      403876
      Max RTT:       839461
      Min RTT:       403876
      Stddev RTT:    185170

So to make it similar to normal ping response changed it to float and ms unit

  Latency:
      Average RTT:   0.724
      Last RTT:      0.635
      Max RTT:       0.857
      Min RTT:       0.635
      Stddev RTT:    0.095

Reply:
OK - but it was already approved in the previous form so now need re-review.